badge.html

```diff
@@ -0,0 +1,175 @@
+                    <a href="avatar.html" class="component-navbar-item-link">Avatar</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">badge</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="alert.html" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="button.html" class="component-navbar-item-link">button</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                
+                
+            
+            </ui>
+        </section>
+
+        <section class="component-doc">
+            <h1 class="component-doc-no">02</h1>
+            <!-- badge  -->
+            <h1 class="component-navbar-heading mg-top20">badge</h1>
+            <h3 class="component-navbar-item">Badges are used to highlight an item's status for quick recognition.</h3>
+
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <span class="badge badge-success">success</span>
+                <span class="badge badge-error">error</span>
+                <span class="badge badge-warning">warning</span>
+                <span class="badge badge-info">info</span>
+            </div>
+            <div class="line-hori"></div>
+
+
+            <!-- /* badge on avatar online, ofline, ddnd */ -->
+            <h1 class="component-navbar-heading mg-top20">badge On avatar</h1>
+            <h3 class="component-navbar-item">Badges are used to highlight an item's status for quick recognition.</h3>
+
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-s"> 
+                    <span class="badge-span badge-online"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-md"> 
+                    <span class="badge-span badge-offline"></span>
+                </div>
+                <div class="badge-on-avatar">
+                    <img src="images/60111.jpg" alt="" class="avatar avatar-l">
+                    <span class="badge-span badge-dnd"></span>
```

Comment:
it seems badge fall not on the image. It would be great if it is perfectly set at bottom-left(say) of the image.

Reply:
done!!